com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs

```diff
@@ -13,6 +13,9 @@ public static class CoreEditorConstants
     /// <summary>Class containing style definition</summary>
     public static class CoreEditorStyles
     {
+        /// <summary>Standard UI spacing</summary>
+        public static float standardSpacing => EditorGUIUtility.standardVerticalSpacing;
```

Comment:
I am not sure to understand why we need this new field?

Reply:
I didn't want to use something for horizontal spacing named vertical

Reply:
Actually, I think vertical and horizontal are not the same value. Vertical is currently about 2px and horizontal is more like 5px in most of inspector.
Also, I am not sure we want this "horizontal" to be changed if the vertical spacing is changed in Unity.
Something like this seams more adequate.
```suggestion
        public static float standardHorizontalSpacing => 5f;
```